packages/extras/src/lib/hooks/useAudioListener.ts

```diff
@@ -1,5 +1,5 @@
 import type { AudioListener } from 'three'
-import { useThrelteAudio } from './useThrelteAudio'
+import { useThrelteAudio } from '@threlte/core'
```

Comment:
The audio context and its hook should be moved to extras as well.

Reply:
The problem then is: what component is providing that context? Will there be a component `<Extras>` that you wrap your entire scene in that provides all necessary contexts used in `@threlte/extras`? Is this a use case for a root plugin?